modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewTest.java

```diff
@@ -2047,8 +2051,12 @@ public TableCell<Person, String> call(TableColumn<Person, String> p) {
         sl.dispose();
     }
 
-//    @Ignore("Test started intermittently failing, most probably due to RT-36855 changeset")
-    @Test public void test_rt_34493() {
+    /**
```

Comment:
We can remove this comment. If needed, we can get this information from looking at file revision history.